src/serlo-editor/editor-ui/side-toolbar-and-wrapper.tsx

```diff
@@ -5,44 +5,30 @@ import { tw } from '@/helper/tw'
 
 export interface SideToolbarAndWrapperProps {
   children: React.ReactNode // The rendered document
-  sideToolbarRef: React.RefObject<HTMLDivElement> // The rendered toolbar buttons
-  hasSideToolbar: boolean // `true` if the document has rendered any toolbar buttons
   renderSideToolbar?(children: React.ReactNode): React.ReactNode // Render prop to override rendering of toolbar
+  sideToolbarRef: React.RefObject<HTMLDivElement> // The rendered toolbar buttons
   focused: boolean // `true` if the document is focused
-  isInlineChildEditor: boolean
+  noSidebar: boolean
 }
 
 // Container that includes a plugin and its sideToolbar and handles some hover&focus styling
 export function SideToolbarAndWrapper({
-  focused,
   children,
   renderSideToolbar,
   sideToolbarRef,
-  hasSideToolbar,
-  isInlineChildEditor,
+  focused,
+  noSidebar,
 }: SideToolbarAndWrapperProps) {
   const [hasHover, setHasHover] = useState(false)
 
-  const showToolbar = hasSideToolbar || renderSideToolbar !== undefined
+  const showToolbar = renderSideToolbar !== undefined
 
   const isFocused = focused && showToolbar
   const isHovered = hasHover && showToolbar
```

Comment:
Super minor, but these names bother me a little bit.

```suggestion
  const isFocused = focused && !!renderSideToolbar
  const isHovered = hasHover && !!renderSideToolbar
```

Or at least change the name of `showToolbar` to `showSideToolbar`.

Reply:
hope to remove this completely with the next PR 👍